k8s/deploy_to_k8s.md

```diff
@@ -4,11 +4,38 @@
 
 Make sure you have Spring Cloud Gateway for k8s installed.
 
+The JSON Web Key Set (JWKS) is a set of keys containing the public keys used to verify any JSON Web Token (JWT) issued by the authorization server and signed using the RS256 signing algorithm.
+
 For Animal Rescue sample Single Sign-On (SSO) to work, you will need to provide two secret files:
+* ./k8s/base/secrets/sso-credentials-for-backend.txt
+* ./k8s/overlays/sso-secret-for-gateway/secrets/test-sso-credentials.txt
+
+Before you start, and for validation, please locate the JWKS endpoint info `jwks_uri` from your provider.
+The endpoint exists at:
+```
+https://YOUR_DOMAIN/.well-known/openid-configuration
+
+# ex: for Okta, for the configured Issuer URI, you can retrieve it at:
+https://<issuer-uri>/.well-known/openid-configuration
+
+$ curl https://dev-1234567.okta.com/oauth2/abcd12345/.well-known/openid-configuration
+
+{
+  "issuer": "https://dev-1234567.okta.com/oauth2/abcd12345",
+...
+  "jwks_uri": "https://dev-1234567.okta.com/oauth2/abcd12345/v1/keys",
+....
+
+# Please note that the format used is jwks_uri="<issuer-uri>/v1/keys"
```

Comment:
This line might be confusing for users not using Okta. Could you add `format used by Okta` or remove this line?

Reply:
yes, one minute

Reply:
done